db/builder.cc

```diff
@@ -136,7 +137,9 @@ Status BuildTable(
       bool use_direct_writes = file_options.use_direct_writes;
       TEST_SYNC_POINT_CALLBACK("BuildTable:create_file", &use_direct_writes);
 #endif  // !NDEBUG
-      IOStatus io_s = NewWritableFile(fs, fname, &file, file_options);
+      FileOptions cur_file_opts = file_options;
+      cur_file_opts.handoff_checksum_type = ChecksumType::kCRC32c;
```

Comment:
Can this be set in the ```FileOptions``` constructor?

Reply:
> Can this be set in the `FileOptions` constructor?

Yes, I will add a new constructor for FileOptions to set the ChekcsumType.